jbpm-examples/pom.xml

```diff
@@ -94,6 +94,10 @@
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-test</artifactId>
     </dependency>
+      <dependency>
+          <groupId>org.jbpm</groupId>
+          <artifactId>jbpm-test-util</artifactId>
+      </dependency>
```

Comment:
Is it really needed here? I don't see any Pooling Data Source in this module. But maybe it is here because of something else...